modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/Amount.scala

```diff
@@ -11,15 +11,13 @@ import eu.timepit.refined._
 import eu.timepit.refined.api.Refined
 import eu.timepit.refined.auto._
 import eu.timepit.refined.cats._
-import eu.timepit.refined.numeric.Interval.OpenClosed
-import eu.timepit.refined.numeric.{NonNegative, Positive}
+import eu.timepit.refined.numeric.{Negative, NonNegative, Positive}
 import eu.timepit.refined.predicates.all.Or
 import io.circe.refined._
 import io.estatico.newtype.macros.newtype
 
 object amount {
-  type NegatedPositive = OpenClosed[W.`Long.MinValue`.T, -1L]
-  type AmountValuePredicate = Positive Or NegatedPositive
+  type AmountValuePredicate = Positive Or Negative
```

Comment:
Does Negative exclude 0?

Reply:
@marcinwadon No. If you drill into both of those two they're defined as `Greater[0]` and `Less[0]`. Maybe I didn't see it but I couldn't find a "NonZero" refinement, which is what this is.

Reply:
Update. I mean YES... it excludes zero. I first read "include"